contract_tests/go/auth-api/support/start-local-auth.sh

```diff
@@ -1,12 +1,3 @@
 #!/bin/bash
-
 set -x
-
-export GOPATH="$(pwd)/local-gopath"
-mkdir -p "$GOPATH"
-
-go get -u github.com/fabric8-services/fabric8-auth
```

Comment:
Wait - is this what you intended? Without this line - the auth service is not downloaded and then cannot be built.

Reply:
@ldimaggi  Yes, this is intended... it's only a PoC/Demo to demonstrate the contract tests. 

So earlier, the auth service (as a provider) was downloaded from sources (incl. all dependencies), built and then started - which is demanding.

While with this change (replacing the get-and-make approach) the auth service is simply started from a pre-built docker image. Thus, it is more simple and less resource-hungry.

Of course, in the real test that will come to the [fabric8-auth](https://github.com/fabric8-services/fabric8-auth) repo, the `make dev` approach will be used.